### ISSUE TITLE
 Problem when I add two components in one comportment

### ISSUE THREAD

**Issue by mehdi550u** [creativetimofficial/material-dashboard-pro-angular#98]
Hi, I have a problem when I add two components in one comportment. Some times just it shows one of them. 
same as this: 
```
<div> 
<app-com1></app-com1> 
<app-com2></app-com2> 
</div>
```
 Some times just shows app-com1, some times shows app-com2, and if I refresh my page using browser refresh button, it shows both

**Comment by alexandru-paduraru** [comment]
@mehdi550u thank you for using our product. Can you please let us know exactly what code you have and what triggers the show/hide of the components? My colleague @chelaruc will have a look after you give us all the details.

Best,
Alex

**Comment by mehdi550u** [comment]
@alexandru-paduraru I had this problem with very simple and basic code,  but this problem resolved after ver 1.4.3.
